src/components/ListToolbox/ListToolbox.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Button } from '../Buttons/Button';
+import styles from './ListToolbox.scss';
+
+type Selection = {
+    amount: number;
+    label: string;
+};
+
+type Toggle = {
+    label: string;
+    onClick: () => void;
+};
+
+type Toggles = {
+    current: 'all' | 'none';
+    all: Toggle;
+    none: Toggle;
+};
+
+type Action = {
+    label: string;
+    disabled: boolean;
```

Comment:
I think the disabling button should lean to the global component `ListToolbox` state (isActive or has items) when actions should be enabled. Or maybe your idea was to activate the action in a separate way when action meets min-max requirements, right?

Reply:
That was my first intent as well, but then I realized that most likely we'll run into situations when we will have to have some actions enabled and some disabled at the same time, so I decided to make it bit more flexible here.